docs/eventing/debugging/README.md

```diff
@@ -341,11 +684,11 @@ kubectl --namespace knative-sources get pod -l control-plane=controller-manager
 This is actually a single binary that runs multiple Source Controllers,
 importantly including [ContainerSource Controller](#containersource-controller).
 
-The `KubernetesEventSource` is fairly simple, as it delegates all functionality
+The `ApiServerSource` is fairly simple, as it delegates all functionality
 to an underlying [ContainerSource](#containersource), so there is likely no
```

Comment:
I don't think this is accurate any longer (although I am looking at head, so it may have still been accurate as of 0.6).

Reply:
The ApiServerSource (new in 0.6) is not based on ContainerSource.